docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 Sphinx==1.7.8
 sphinx-argparse==0.2.2
 sphinxcontrib-websupport==1.1.0
+sphinx_rtd_theme
```

Comment:
This is a duplicate of https://github.com/awslabs/aws-config-rdk/blob/08df139b0a2e986ef56e2d5bd4dd6ec318f5fce9/requirements-dev.txt#L2

Reply:
Is there a reason for the requirements-dev.txt considering it contains only doc libraries which are covered in docs/requirements.txt? Thinking in the name of consistency, it's probably worth putting all doc requirements in one place?

Reply:
@jack1902 I would love to have requirements.txt replacing requirements-dev.txt. As an open source, we should be more generic with no environment names defined for users.

Reply:
Since the `requirements` for the docs is in the `docs` folder i see no reason to keep the `-dev` file. Might be worth updating the docs on how to get the docs running locally as i never noticed the `requirements` in the `docs` folder before 😄

Reply:
will update the PR to remove the `requirements-dev.txt`

On another related question, should we lock the version or not? I am sure all the other dependencies will lock `sphinx-rtd-theme` but do we want to be more explicit?

Reply:
lock the theme to the latest currently available, might be worth just seeing how it works with the latest for all the requirements listed in there. Never hurts to try to keep things up to date